lib/themes/dosomething/paraneue_dosomething/scss/patterns/_photo.scss

```diff
@@ -22,34 +26,34 @@
     img {
       width: 100%;
     }
+  }
+}
 
-    .__copy {
-      font-size: $font-smaller;
-      height: 110px;
-      line-height: 1.2;
-      overflow: hidden;
-      margin-top: ($base-spacing / 2);
-      word-wrap: break-word;
-
-      .photo__caption {
-        font-size: $font-smaller;
-      }
-
-      .footnote {
-        font-size: $font-small;
-      }
-
-      .form-actions {
-        margin-top: 6px;
-      }
-
-      @include media($large) {
-        font-size: $font-small;
-      }
-    }
+.photo__copy {
+  font-size: $font-smaller;
+  height: 110px;
+  line-height: 1.2;
+  overflow: hidden;
+  margin-top: ($base-spacing / 2);
+  word-wrap: break-word;
+
+  .photo__caption {
+    font-size: $font-smaller;
   }
 
-  .photo--actions {
-    width: 100%;
+  .footnote {
+    font-size: $font-small;
   }
+
+  .form-actions {
```

Comment:
Yep, I haven't touched it since writing it for the original prototype. That reduces the top padding on the `form-actions` that's used for the kudos... I wonder if we could remove it without borking the design. Or if not, it should probably be moved as part of `.photo__actions` rather than a specificity-based override.